test/unit/guard_clause_test.exs

```diff
@@ -37,4 +37,24 @@ defmodule ExFormat.Unit.GuardClauseTest do
       """
     end
   end
+
+  test "indentation for multiline guard clauses should be consistent" do
+    assert_format_string """
+    defmacrop f(a)
+              when is_atom(a) and
+                a in @test1 and
```

Comment:
this should be indented same as "is_atom". I would say it's because they are all args to when, wdyt @josevalim?